src/ProjectCanineDatabase/Tables/Tests.sql

```diff
@@ -3,7 +3,6 @@
 	[Id] INT IDENTITY(1,1) PRIMARY KEY,
 	[ShortName] VARCHAR(30) NOT NULL,
 	[Name] VARCHAR(500) NOT NULL,
-	[TestVersion_DateTime] DATETIME NOT NULL,
-	[LastEditedBy] INT NOT NULL,
-	[LastEditedDate] DATETIME NOT NULL
+	[LastEditedBy] VARCHAR(255) NOT NULL,

```

Comment:
shouldn't be varchar, this is a foreign key

Reply:
FK to what? There is no user table and hopefully never will be since we should use Azure AD. The only purpose of that col is to provide a human-readable account of the user who edited it. We should keep it simple and just store the username.

Reply:
Just saw the earlier remark about Examiners. If an Examiner is the only type of person who can use the app to edit tests and questions, then I guess that makes sense.